### ISSUE TITLE
[Plugin][Enh] Sensitivity analysis: implement more methods

### ISSUE THREAD

**Issue by JusteRaimbault** [openmole/openmole#480]
See Cosenza, A., Mannina, G., Vanrolleghem, P. A., & Neumann, M. B. (2013). Global sensitivity analysis in wastewater applications: A comprehensive comparison of different methods. Environmental modelling & software, 49, 40-52.
for an other Global Sensitivity Analysis method: Standardized Regression Coefficient (basically do a linear regression on runs done on a LHS sampling) - would be good to implement it

**Comment by JusteRaimbault** [comment]
Plus a normalized index for Saltelli Variance-based indices: S_Ni = (S_Ti - S_i) / max(S_Ti - S_i) should be easy to add to `SensitivitySaltelli`

**Comment by samthiriot** [comment]
Hello,
a side note: it is annoying not to have a sensitivity method able to deal with categorial inputs.
S.

**Comment by JusteRaimbault** [comment]
True, it would be useful! Do you have a particular method in mind? There seems to exist a few but I did not find any close to Global Sensitivity Analysis methods

**Comment by JusteRaimbault** [comment]
In relation with the Standardized Regression Coefficients methods, more elaborated surrogate models can be used - see Shahsavani, D., & Grimvall, A. (2011). Variance-based sensitivity analysis of model outputs using surrogate models. Environmental Modelling & Software, 26(6), 723-730.
@francoismaths is working on a similar approach since the coding camp
